docs/_ext/napdocs.py

```diff
@@ -1,6 +1,7 @@
 """
 Sphinx plugins for nap documentation.
 """
+from __future__ import unicode_literals
```

Comment:
It's too bad they both need to be at the top of every file, it's so messy.
:(

On Fri, Nov 13, 2015 at 9:20 AM, Joseph Bane <notifications@github.com>
wrote:

> In docs/_ext/napdocs.py
> <https://github.com/CashStar/nap/pull/9#discussion_r44786100>:
>
> > @@ -1,6 +1,7 @@
> >  """
> >  Sphinx plugins for nap documentation.
> >  """
> > +from __future__ import unicode_literals
>
> I'd like to see absolute_imports as well if you're adding this much.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CashStar/nap/pull/9/files#r44786100>.
>